process-migration-service/pom.xml

```diff
@@ -2,549 +2,630 @@
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-lang3</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>mysql</groupId>
-            <artifactId>mysql-connector-java</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>com.h2database</groupId>
-            <artifactId>h2</artifactId>
-        </dependency>
+    <!-- PIM service configuration -->
+    <pim.server.http.rest>http://${container.hostname}:8180/rest</pim.server.http.rest>
+    <pim.server.http.health>${pim.server.http.rest}/health/readiness</pim.server.http.health>
+    <pim.username>kermit</pim.username>
+    <pim.password>thefrog</pim.password>
+  </properties>
 
-        <!-- Test -->
-        <dependency>
-            <groupId>org.jboss.weld</groupId>
-            <artifactId>weld-junit4</artifactId>
-            <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <artifactId>hibernate-jpa-2.1-api</artifactId>
-                    <groupId>org.hibernate.javax.persistence</groupId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.hibernate</groupId>
-            <artifactId>hibernate-entitymanager</artifactId>
-            <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>javax.activation</groupId>
-                    <artifactId>javax.activation-api</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>io.thorntail</groupId>
-            <artifactId>arquillian</artifactId>
-            <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <artifactId>thorntail-jdk-specific</artifactId>
-                    <groupId>io.thorntail.jdk-specific</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>jboss-modules</artifactId>
-                    <groupId>org.jboss.modules</groupId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.kie.soup</groupId>
-            <artifactId>kie-soup-maven-integration</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.kie.server</groupId>
-            <artifactId>kie-server</artifactId>
-            <classifier>${kie.server.classifier}</classifier>
-            <type>war</type>
-            <scope>test</scope>
-        </dependency>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.thorntail</groupId>
+        <artifactId>bom-all</artifactId>
```

Comment:
Why we need here to import `io.thorntail:bom-all` instead of just `io.thorntail:bom` ?

Reply:
To be honest, I don't know. Maybe it would work also without it.